### PR TITLE
2024 6 2

### PR DESCRIPTION
**Pull request by ntoll**

### CODE REVIEW

File: docs/user-guide/builtins.md [hunk 1]
```diff
@@ -372,6 +372,80 @@ The following code demonstrates a `pyscript.WebSocket` in action.
...
+        window.console.log(key, value)
+        ...
+
+
+store = await storage("my-data-store", storage_class=MyStorage)
+
+# The store object is now an instance of MyStorage.
+```
+
+!!! warning
```
Comment (WebReflection) [inline docs/user-guide/builtins.md:441]: I just wonder if this should be moved at the beginning, not at the end ... no strong feelings round it, just my surprise to find it even after the *extend* advanced use case.

Comment (WebReflection) [reply docs/user-guide/builtins.md:441]: or can we have at least a top level link to this part where we explain **where** the persistent data exists?

**Review by WebReflection — APPROVED** [review]
There's a minor (?) hint around an important topic for users (imho) but otherwise this looks good to me!

File: docs/user-guide/builtins.md (hunk 1 above)
Comment (ntoll) [reply docs/user-guide/builtins.md]: I've moved it to the top of the section. :+1:

**Pull request merged**